eldritch/events.py

```diff
@@ -2500,8 +2503,9 @@ def choice_count(self):
 
 class CombatChoice(ItemChoice):
 
-  def __init__(self, character, prompt):
+  def __init__(self, character, prompt, combat_round=None):
```

Comment:
I'm not in love with this as a feature. It's optional because there are a couple encounters that directly create CombatChoice's without a related CombatRound.